theme-data/dark/dark-common.json

```diff
@@ -186,6 +186,23 @@
       "primary": "@color-black-alpha-90",
       "secondary": "@color-black-alpha-40"
     },
+    "reaction": {
```

Comment:
This file (and the below) should contain only things in https://www.figma.com/file/yYfvFEEr3UkW1yUKkW7zEH/Tokens---Theme-Color?node-id=0%3A1